### ISSUE TITLE
Windows 10: TeXworks hangs indefinitely after opening a file for the third time

### ISSUE THREAD

**Issue by matteocoder** [TeXworks/texworks#891]
**Bug description:**
TeXworks will not start anymore if it has opened a `.tex` file for three times.
It will just stay as a background process using half of the CPU.

**Steps to reproduce the problem:**

1. Open a `.tex` file with TeXworksup to two times
2. Kill the `texworks.exe` process
3. Try to open the `.tex` file a third time 

**Expected behavior:**
TeXworks should open the file.

**General information:**
_TeXworks version_:  0.6.3
_TeXworks obtained from_: both as part of TeXLive 2019 and installed directly from `www.tug.org`
_Operating system_:  Windows 10 Pro 1909 64-bit

**Additional information:**
I installed TeXLive 2019 last summer for the first time, and TeXworks worked just fine,
but then I uninstalled the distributionand I reinstalled it some time later.
After the reinstallation this bug appeared:
after opening a `.tex` file for the second time, TeXworks would hang indefinitely and use half of the CPU, then if I tried to open the file again, the program just would not startand again use half of the CPU in the background. If I killed the `texworks.exe` process,Windows would return this error:
```
C:\texlive\2019\bin\win32\runscript.tlu:907: command failed with exit code 1:texworks "C:\path\to\the\file\I\was\trying\to\open"
```
I had installed TeXworks as part of TeXLive,
so I thought that installing it as a standalone program might solve the problem.
Unfortunately, the same thing happened after using the new installation for three times,
but then if I tried to kill the `texworks.exe` process, there would be no error.

Unfortunately, [the solution provided by this answer](https://tex.stackexchange.com/questions/488208/texworks-takes-more-than-20-minutes-to-start)
does not seem to work for me.

Thank you for your attention and have a nice day.

**Comment by stloeffler** [comment]
Thanks for the report. I have a few additional questions:
1. Do you mean TeXworks would hang if you open three different .tex files (e.g., `a.tex`, `b.tex`, `c.tex`) or if you are trying to open the same file three times?
2. Am I correct in assuming that TeXworks behaved correctly as long as only one document was opened (once)?
3. Where were the files located? On your hard disk, a network drive, an external drive (e.g. a USB stick)? Were any of them in a folder that was synchronized to a cloud (e.g. Dropbox)?
4. How did you open the files? By double-clicking them in the explorer, or from within TeXworks using File > Open?
5. Were there associated .pdf files? If so, were they corrupted? Were they opened?
6. Could you also test this with the [latest development version of TeXworks](https://bintray.com/texworks/Windows-latest/TeXworks-for-Windows%3Alatest/view)?

Just for reference, it should be impossible to open the same file twice (i.e., in two different windows) - the second time you open an already opened file, TeXworks should just show you the already open window.

**Comment by matteocoder** [comment]
@stloeffler Thank you for your reply.
1. I mean TeXworks would hang if I tried to open *any* `.tex` file after the third time that the program has been launched since the first installation.
2. Unfortunately no, because I can't open documents at all.
3. The file is located on my computer's internal hard drive, but it resides on the D: partition instead of the C: partition (both partitions are in the same drive), and it is not synchronized with any cloud service. Also, the file is inside a directory that is symlinked to my user directory inside the C: partition.
4. I tried to open the file from the Explorer, though TeXworks would hang even if I tried to just launch the program itself from the Start menu.
5. Yes, there were `.pdf` files, they were not corrupted and they were closed.
6. I downloaded version 0.7-202003040940-git_dd36b12, extracted it to my user directory and launched the TeXworks: opening files from within TeXworks works fine, even after launching the program multiple times.
